fluent-syntax/makefile

```diff
@@ -23,6 +23,10 @@ compat.js: $(SOURCES)
 	    --output.file $@
 	@echo -e " $(OK) $@ built"
 
+.PHONY: test-without-babel-polyfill
+test-without-babel-polyfill:
+	@mocha --recursive --ui tdd --require babel-register test/
```

Comment:
We still need `babel-register` to transpile ES6 syntax, just like the compat builds do. Thanks to this, `import`s work correctly, for instance.